website/screens/components/file-input/code/FileInputCodePage.tsx

```diff
@@ -24,19 +24,6 @@ const sections = [
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>
-              <DxcFlex direction="column" gap="0.25rem" alignItems="baseline">
-                <StatusBadge status="deprecated" />
-                name
-              </DxcFlex>
-            </td>
-            <td>
-              <TableCode>string</TableCode>
-            </td>
```

Comment:
We can remove `name` from props also. If it requires some changes we can do it in another PR, but it seems not.